src/ts/mapboxMap.ts

```diff
@@ -4,7 +4,7 @@ module powerbi.extensibility.visual {
     declare var MapboxDraw : any;
 
     export class MapboxMap implements IVisual {
-        private map: mapboxgl.Map;
+        private map: any;
```

Comment:
Why does it need to be `any`? I'm not that big of a fan of Typescript so I'm just curious if this change had some reason behind it.

Reply:
Nevermind I guess I see now. :)

Reply:
@szilardhuber good catch, this was a debug fix I tried before updating the `typings` for mapboxgl-js to include the `api_config` option.  It's not required - removing.